essentials-framework/Essentials Core/PepperDashEssentialsBase/File/FileIO.cs

```diff
@@ -0,0 +1,163 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Crestron.SimplSharp;
+using Crestron.SimplSharp.CrestronIO;
+using PepperDash.Core;
+
+namespace PepperDash.Essentials.Core
+{
+	public static class FileIO
+	{
+
+		static CCriticalSection fileLock = new CCriticalSection();
+		/// <summary>
+		/// 
+		/// </summary>
+		/// <param name="fileName"></param>
+		/// <returns></returns>
+		public static bool FileExsists(string fileName)
```

Comment:
Silly typo.  `FileExists(string fileName)`

It might be more efficient to use the .NET `File.Exists()` from `Crestron.SimplSharp.CrestronIO` namespace